src/plugin.js

```diff
@@ -309,9 +324,17 @@ const plugin = function(options) {
   // because it doesn't make sense to pass it to each Overlay component.
   delete settings.overlays;
 
-  this.overlays_ = overlays.map(
-    o => this.addChild('overlay', videojs.mergeOptions(settings, o))
-  );
+  this.overlays_ = overlays.map(o => {
+    let mergeOptions = videojs.mergeOptions(settings, o);
+
+    // Attach bottom aligned overlays to the control bar so
+    // they will adjust positioning when the control bar minimizes
+    if (this.controlBar && mergeOptions.align.indexOf('bottom') !== -1) {
```

Comment:
That's reasonable for the future but people would potentially break if this is a minor update, assuming that it isn't an option.